nucypher/utilities/prometheus/collector.py

```diff
@@ -30,6 +29,7 @@
 from nucypher.blockchain.eth.interfaces import BlockchainInterfaceFactory
 from nucypher.blockchain.eth.registry import BaseContractRegistry
 
+from prometheus_client.metrics import MetricWrapperBase
```

Comment:
Do you want to go as far as wrapping this for `ImportError` and re-raising `DevelopmentInstallationRequired`?

Reply:
Do you mean on L21 - the check is performed there, this is just an additional import.

We could throw a specific exception or perhaps just provide a more specific message. However, `DevelopmentInstallationRequired` might not tell the whole story since it is not solely a dev installation problem because `pip install nucypher[ursula]` would install `prometheus_client`.

Reply:
True, and since it's only a single dependency it's not worth introducing specific guidance?

Reply:
I guess the real goal for `pip install nucypher[ursula]` is to install optional ursula packages - currently only `prometheus`? I only ask because the ursula directive only installs prometheus so we could just add that to the message - but will the list of what get installed for that directive be added to over time?

An alternative is to point to prometheus docs (yet to be written 😅). 

Perhaps, for now, we just add the `pip install nucypher[ursula]` to skip the extra bloat of `dev` - wdyt? Although docs should still be written haha.

Reply:
> I guess the real goal for pip install nucypher[ursula] is to install optional ursula packages
:+1: - Yep - tying to scope the package as thin and purpose-built as possible, this being the first step.

> Perhaps, for now, we just add the pip install nucypher[ursula] to skip the extra bloat of dev - wdyt? Although docs should still be written haha.

Yes, and yes. :-)

Reply:
✔️